### ISSUE TITLE
Screen reader accessibility issue with NgbDatepicker

### ISSUE THREAD

**Issue by pradeepKaatnam-ms** [ng-bootstrap/ng-bootstrap#3735]
### Bug description:
We are facing **screen reader accessibility issue** with **NgbDatepicker** in our project. The Month and Year controls(input boxes) should have proper accessibility name so that the screen reader users would be able to use those controls.

### StackBlitz that reproduces the issue:

https://stackblitz.com/edit/angular-kbbygx?file=src%2Fapp%2Fapp.component.html

We are using a tool called **Accessibility Insights for Windows** to identify these kind of issues. I am attaching a screenshot where the name property of Month and year controls are empty.

![image](https://user-images.githubusercontent.com/65524145/82200325-7805c700-991c-11ea-92ff-de705f60e2e3.png)

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 4.0.0

ng-bootstrap: 2.0.0

Bootstrap: 4.10

### Expected
Correct Name should be given for Month and Year control present on the Calendar.
For Ex: Name for "Month" combo box should be present as "Month" and similarly for Year combo box as well.

**Comment by IAfanasovMob** [comment]
Hi @pradeepKaatnam-ms ! 
I believe the answer would be the same as for [3734](https://github.com/ng-bootstrap/ng-bootstrap/issues/3734).
You could fork the repo and do adjustment yourselves or try to hire a consultant to do it for you if you really want to use an old version of the library.

**Comment by pradeepKaatnam-ms** [comment]
Hi @IAfanasovMob 
You mean to say the same got fixed in the latest versions of bootstrap?

**Comment by IAfanasov** [comment]
Yes, looking into the [latest source code](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/datepicker/datepicker-navigation-select.ts) lines 26,27 and 34,35 I clearly see proper attributes for screen readers.

**Comment by pradeepKaatnam-ms** [comment]
Yep. Got it. Thanks @IAfanasov for your quick response. I will close this bug.
